### DISCUSSION TITLE
Phaser v3.85.2

### DISCUSSION THREAD

**Discussion by photonstorm** [photonstorm/phaser#6906] [Announcements]
## Updates

* `WebGLRenderer.setExtensions` is a new method that queries the GL context to get the list of supported extensions. Which it then sets into the class properties. This method is called internally as part of the `init` and restore process.

## Bug Fixes

* When the WebGL context was restored it would incorrectly try to call `init.setupExtensions()` which didn't exist. It now calls the correct method, `WebGLRenderer.setExtensions`. Fix #6905 (thanks @RedRoosterMobile)
* `TransformMatrix.setQuad` has been fixed so it no longer rounds the quad dimensions, only the x/y coordinates. This fixes a bug where it could give slightly different (+- 1px) sized textures based on how the dimensions were rounded when using `roundPixels` on the camera. Fix #6874 (thanks @saintflow47)

## Examples, Documentation, Beta Testing and TypeScript

Thanks to the following for helping with the Phaser Examples, Beta Testing, Docs, and TypeScript definitions, either by reporting errors, fixing them, or helping author the docs:

@AlvaroNeuronup

<hr /><em>This discussion was created from the release <a href='https://github.com/phaserjs/phaser/releases/tag/v3.85.2'>Phaser v3.85.2</a>.</em>

**Comment by codingthat** [comment]
Hey folks, great work as usual.

At the bottom of the last two sets of [release](https://github.com/phaserjs/phaser/discussions/6902) [notes](https://github.com/phaserjs/phaser/discussions/6903), it's noted that:

> We are removing Phaser.Struct.Set and replacing it with a regular JS Set instance. This means methods like iterateLocal will be gone.

Is that still planned?

Cheers :)

**Reply by photonstorm** [reply]
This is still planned for the v4 release, yes. But not for any v3 release.
